Database Schema has Overflow

Check it out there is an [overflow](https://github.com/tijko/SportsWareHouse/actions/runs/8045267283/job/21970229731#step:7:2452) hitting the the Test-Actions....

```
 test.py:42: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/cursors.py:153: in execute
    result = self._query(query)
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/cursors.py:322: in _query
    conn.query(q)
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/connections.py:558: in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/connections.py:822: in _read_query_result
    result.read()
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/connections.py:1200: in read
    first_packet = self.connection._read_packet()
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/connections.py:772: in _read_packet
    packet.raise_for_error()
/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/protocol.py:221: in raise_for_error
    err.raise_mysql_exception(self._data)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

data = b"\xff~\x05#22001Data too long for column 'table_name' at row 1"

    def raise_mysql_exception(data):
        errno = struct.unpack("<h", data[1:3])[0]
        errval = data[9:].decode("utf-8", "replace")
        errorclass = error_map.get(errno)
        if errorclass is None:
            errorclass = InternalError if errno < 1000 else OperationalError
>       raise errorclass(errno, errval)
E       pymysql.err.DataError: (1406, "Data too long for column 'table_name' at row 1")

/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/pymysql/err.py:143: DataError
=========================== short test summary info ============================
ERROR test.py::test_mysql_data - pymysql.err.DataError: (1406, "Data too long for column 'table_name' at row 1")
=============================== 1 error in 0.81s ===============================
```

--- a/Test/test.py
+++ b/Test/test.py
@@ -48,6 +48,20 @@ def mysql_db():
     cursor.close()
     connection.close()
 
+def test_max_word(mysql_db):
+    cursor = mysql_db.cursor()
+    item = 'pneumonoultramicroscopicsilicovolcanoconiosis'
+    print('Testing max word ({}) insert into field.'.format(item))
+    item_number = data_hash[item]
+    print('Inserting max word ({}) into database....'.format(item))
+    query = "INSERT INTO my_table(table_name, table_number) VALUES (%s, %s)"
+    cursor.execute(query, (item, item_number,))
+    query = "SELECT * FROM my_table WHERE table_number = %s"
+    print('Querying database....')
+    cursor.execute(query, (item_number))
+    test_fetch = cursor.fetchone()
+    assert item == test_fetch['table_name']
+
 def test_mysql_data(mysql_db):
     # Use the mysql_db fixture to interact with the MySQL database
     cursor = mysql_db.cursor()

--- a/Test/test.py
+++ b/Test/test.py
@@ -19,7 +19,6 @@ def mysql_db():
     )
     cursor = connection.cursor()
     cursor.execute("CREATE TABLE IF NOT EXISTS my_table(table_id INT PRIMARY KEY AUTO_INCREMENT, table_name VARCHAR(64) NOT NULL, table_number INT NOT NULL)")
-    global query
     query = "INSERT INTO my_table(table_name, table_number) VALUES (%s, %s)"
     picks = 0
     # Load data
@@ -48,6 +47,20 @@ def mysql_db():
     # Close the cursor and the connection
     cursor.close()
     connection.close()
+
+def test_max_word(mysql_db):
+    cursor = mysql_db.cursor()
+    item = 'pneumonoultramicroscopicsilicovolcanoconiosis'
+    print('Testing max word ({}) insert into field.'.format(item))
+    item_number = data_hash[item]
+    print('Inserting max word ({}) into database....'.format(item))
+    query = "INSERT INTO my_table(table_name, table_number) VALUES (%s, %s)"
+    cursor.execute(query, (item, item_number,))
+    query = "SELECT * FROM my_table WHERE table_number = %s"
+    print('Querying database....')
+    cursor.execute(query, (item_number))
+    test_fetch = cursor.fetchone()
+    assert item == test_fetch['table_name']
 
 def test_mysql_data(mysql_db):
     # Use the mysql_db fixture to interact with the MySQL database

